.github/workflows/prometheus-compliance-tests.yml

```
@@ -45,9 +45,10 @@ jobs:
         with:
           repository: prometheus/compliance
           path: compliance
-          ref: f0482884578bac67b053e3eaa1ca7f783d146557
       - name: Copy binary to compliance directory
-        run: mkdir compliance/remote_write_sender/bin && cp opentelemetry-collector-contrib/bin/otelcontribcol_linux_amd64 compliance/remote_write_sender/bin/otelcol_linux_amd64
+        #The actual name of downloaded artifact at https://github.com/prometheus/compliance/blob/main/remote_write_sender/targets/otel.go#L8 is `otelcol_0.42.0_linux_amd64`,
+        #so we place the collector contrib artefact under the same name in the bin folder to run.
```

```suggestion
        # The required name of the downloaded artifact is `otelcol_0.42.0_linux_amd64`,
        # so we place the collector contrib artifact under the same name in the bin folder to run.
        # Source: https://github.com/prometheus/compliance/blob/12cbdf92abf7737531871ab7620a2de965fc5382/remote_write_sender/targets/otel.go#L8
```
Changed some wording, and made the URL refer to a specific commit in case the file changes in the future.

---

Committed the change, referring to specific commit is good idea certainly. Thanks